src/hyperfine/benchmark.rs

```diff
@@ -358,16 +371,18 @@ pub fn run_benchmark(
 
     let (user_str, user_unit) = format_duration_unit(user_mean, None);
     let system_str = format_duration(system_mean, Some(user_unit));
+    let max_rss_str = format!("{:.3} MB", max_rss_mean);
 
     if options.output_style != OutputStyleOption::Disabled {
         println!(
-            "  Time ({} ± {}):     {:>8} ± {:>8}    [User: {}, System: {}]",
+            "  Time ({} ± {}):     {:>8} ± {:>8}    [User: {}, System: {}, MaxRSS: {}]",
```

Comment:
I'm pretty sure that most users would not know what "MaxRSS" refers to. Maybe just "Mem:" and then explain the precise meaning in the `--help` text and man page?

Reply:
That's a good idea.